### PR TITLE
update: develop localterra and module specs

### PR DESCRIPTION
**Pull request by terran6**
Correcting files in develop/localterra and develop/module-specifications

### CODE REVIEW

File: docs/develop/module-specifications/spec-capability.mdx [hunk 1]
```diff
@@ -53,11 +53,11 @@ func NewApp(...) *App {
...
 
 Capabilities can be claimed by other modules, which add them as owners. `ClaimCapability` allows a module to claim a capability key that it has received from another module so that `GetCapability` calls made later will succeed. If a module that receives a capability wants to access it by name later, `ClaimCapability` must be called. Because capabilities are multi-owner, if multiple modules have one capability reference, all the modules own it. If a module receives a capability from another module but does not call `ClaimCapability`, it may use it in the executing transaction but will not be able to access it afterward.
 
-Any module can call `AuthenticateCapability` to check whether a capability corresponds to a particular name, including un-trusted user input, with which the calling module previously associated it.
+Any module can call `AuthenticateCapability` to check whether a capability corresponds to a particular name, including un-trusted user input, which the calling module previously associated with it.
```
Comment (evanorti) [inline docs/develop/module-specifications/spec-capability.mdx:60]: How can we simplify this? 

We should turn this phrase into 2 separate sentences.

File: docs/develop/module-specifications/spec-distribution.mdx [hunk 2]
```diff
@@ -30,7 +30,7 @@ Collected rewards are pooled globally and distrubuted to validators and delegato
 
 All globally tracked parameters for distribution are stored within
 `FeePool`. Rewards are collected and added to the reward pool and
-distributed to validators/delegators from here.
+distributed to validators/delegators from there.
```
Comment (evanorti) [inline docs/develop/module-specifications/spec-distribution.mdx:33]: Maybe we can make this sentence clearer:

Fees are collected and stored in the `FeePool` until they can be distributed to validators and their delegators.

File: docs/develop/module-specifications/spec-wasm.mdx [hunk 3]
```diff
@@ -61,15 +61,15 @@ The current owner of the smart contract can reassign a new owner to the contract
...
+Users can specify which query function to use alongside any arguments with a JSON `QueryMsg`. Even though there is no gas fee, the query function's execution is capped by gas determined by metered execution, which is not charged, as a form of spam protection.
 
 ### Wasmer VM
 
 The actual execution of WASM bytecode is performed by [wasmer](https://github.com/wasmerio/wasmer), which provides a lightweight, sandboxed runtime with metered execution to account for the resource cost of computation.
 
 #### Gas meter
 
-In addition to the regular gas fees incurred from creating the transaction, Terra also calculates a separate gas when executing smart contract code. This is tracked by the **gas meter**, which is during the execution of every opcode and gets translated back to native Terra gas via a constant multiplier (currently set to 100).
+In addition to the regular gas fees incurred from creating the transaction, Terra also calculates a separate gas when executing smart contract code. This is tracked by the **gas meter** during the execution of every opcode. The result is then translated back to native Terra gas via a constant multiplier (currently set to 100).
```
Comment (evanorti) [inline docs/develop/module-specifications/spec-wasm.mdx:72]: Much easier to read.

**Review by evanorti — APPROVED** [review]

File: docs/develop/module-specifications/spec-capability.mdx (hunk 1 above)
Comment (terran6) [reply docs/develop/module-specifications/spec-capability.mdx]: @evanorti I think "Any module can call `AuthenticateCapability` to check whether a capability corresponds to the name which the calling module previously associated with it." is what they meant?

**Pull request merged**